src/javascripts/components/mobile-navigation.js

```diff
@@ -65,7 +65,7 @@ MobileNav.prototype.bindUIEvents = function () {
 
 MobileNav.prototype.includeAria = function () {
   this.$nav.setAttribute('aria-hidden', 'true')
-  this.$nav.setAttribute('aria-labelledby', 'app-header-mobile-nav-toggler')
+  this.$nav.setAttribute('aria-label', 'Top-level')
```

Comment:
Given that there's only one 'navigation' landmark in the accessibility tree I think this may not be needed at all as the user will know it's the top level since there's only one.